src/plugins/http_gob_operator.py

```diff
@@ -160,11 +163,11 @@ def execute(self, context):
                     # We retry several times
                     query = gql_file.read()
 
+                force_refresh_token = False
```

Comment:
Misschien deze moet in het begin van de loop?

Reply:
Hmm, de diff ziet er een beetje verwarrend uit in github. Dit zelf wel aan het begin van de loop, maar het gaan om de loop die start op regel 167.